index.js

```diff
@@ -1,47 +1,53 @@
 'use strict';
 
-let AWSXRay = require('aws-xray-sdk');
-let url_lib = require('url');
-let https = AWSXRay.captureHTTPs(require('https'));
-let AWS = AWSXRay.captureAWS(require('aws-sdk'));
-let s3 = new AWS.S3();
+const AWSXRay = require('aws-xray-sdk');
+const urllib = require('url');
+const https = AWSXRay.captureHTTPs(require('https'));
+const AWS = AWSXRay.captureAWS(require('aws-sdk'));
+const s3 = new AWS.S3();
 
-let reverse = (s) => {
-  var o = '';
+exports.reverse = (s) => {
+  const trimmed = s.trim();
+  let o = '';
```

Comment:
Codacy has a fix for the issue: [Replace `''` with `""`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
  let o = "";
```